packages/arb-token-bridge-ui/src/hooks/CCTP/useIsCctpAllowed.ts

```diff
@@ -0,0 +1,20 @@
+import useSWRImmutable from 'swr/immutable'
+import { useCCTP, UseCCTPParams } from './useCCTP'
+
+export function useIsCctpAllowed({ sourceChainId }: UseCCTPParams) {
```

Comment:
We don't really need to pass `sourceChainId` though? We can just default it to Mainnet